Workshop.md

```diff
@@ -0,0 +1,479 @@
+   ```swift
+   import KituraOpenAPI
+   ```
+3. Add the following code into the end of the `postInit()` function after the call to `initializeHealthRoutes()`:  
+   ```swift
+   KituraOpenAPI.addEndpoints(to: router)
+   ```
+
+4. Re-run the server project in Xcode:
+    1) Edit the scheme again and select a Run Executable of "ToDoServer". 2) Run the project, then "Allow incoming network connections" if you are prompted.
+
+Now, you can open [http://localhost:8080/openapi](http://localhost:8080/openapi) and view the live OpenAPI specification of your Kitura application in JSON format.
+
+You can also open [http://localhost:8080/openapi/ui](http://localhost:8080/openapi/ui) and view SwaggerUI, a popular API development tool. You will see one route defined: the GET `/health` route you visited earlier. Click on the route to expand it, then click "Try it out!" to query the API from inside SwaggerUI.
+
+You should see a Response Body in JSON format, like:
+
+```
+{
+  "status": "UP",
+  "details": [],
+  "timestamp": "2018-06-04T16:03:17+0000"
+}
+```
+
+and a Response Code of 200.
+
+Congratulations, you have added OpenAPI support to your Kitura application and used SwaggerUI to query a REST API!
+
+### 3. Add Cross Origin Resource Sharing (CORS) Support
+
+Re-run the ToDo-Backend tests by reloading the test page in your browser.
+
+The first test should still fail with the following:
+
+:x: `the api root responds to a GET (i.e. the server is up and accessible, CORS headers are set up)`
+
+```
+AssertionError: expected promise to be fulfilled but it was rejected with [Error: 
+
+GET http://localhost:8080
+FAILED
+
+The browser failed entirely when make an AJAX request.
+Either there is a network issue in reaching the url, or the
+server isn't doing the CORS things it needs to do.
+```
+
+This test is still failing, even though the server is responding on `localhost:8080`. This is because Cross Origin Resource Sharing (CORS) is not enabled.
+
+By default, web servers only serve content to web pages that were served by that web server. In order to allow other web pages, such as the ToDo-Backend test page, to connect to the server, [Cross Origin Resource Sharing (CORS)](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) must be enabled.
+
+Kitura provides a package which makes it easy to enable CORS in your application. Let's add CORS to your project.
+
+1. Open the `ToDoServer` > `Package.swift` file
+2. Add the following to the end of the dependencies section of the Package.swift file:
+    ```swift
+        .package(url: "https://github.com/IBM-Swift/Kitura-CORS.git", from: "2.1.0"),
+    ```
+3. Update the target dependencies for the "Application" target to the following (note the lack of hyphen in KituraCORS):
+    ```swift
+    .target(name: "Application", dependencies: ["KituraCORS", "KituraOpenAPI", "Kitura", "Configuration", "CloudEnvironment", "Health", "SwiftMetrics" ]),
```

Comment:
There is a reference to "Configuration" which shouldn't be here (I notice it was removed from the target above though).